dataprep/eda/distribution/compute/__init__.py

```diff
@@ -59,28 +59,46 @@ def compute(
     """
     # pylint: disable=too-many-arguments
 
-    df = to_dask(df)
-    df.columns = df.columns.astype(str)
-    df = string_dtype_to_object(df)
+    params = []
+    for temp in (x, y, z):
+        if is_dtype(temp, Geopoint()):
+            name, lat_long = concat_latlong(df, temp)
+            df[name] = lat_long
```

Comment:
Do not directly modify the original dataframe. Besides, the hard-coded `name` may not work, consider the following two cases: 1) the original dataframe already contains the `name` column (lat_long in this case).  2) more than 1 variable from [x,y,z] are GeoPoint.